README.md

```diff
@@ -128,6 +129,15 @@ ALTER SESSION SET SCRIPT_LANGUAGES='<LANGUAGE_ALIAS>=localzmq+protobuf:///<bucke
 * Can push Docker images to Docker registries
 * Run tests for you container against an Exasol DB (docker-db or external db)
 
+## Limitations
+
+* Caution with symbolic links inside flavor path: 
+  If you use symbolic links inside the flavor path (--flavor-path)
+  they must not point to directories outside the root of the flavor path. 
+  Background: Local directories paths must be mounted manually to the docker container. 
```

Comment:
Maybe also add that it is rare and it only affects symlinks outside of the current working directory, because we mount this per default. And, I think you can create a ticket, such that we add an Environment variable which allows you to specifiy additional directories.

Reply:
Instead of managing this manually with an Env variable, wouldn't it be better to traverse the directories and check for any symbolic link?

Reply:
Created #35 for this issue.